src/org/labkey/test/components/ui/grids/EditableGrid.java

```diff
@@ -1155,6 +1156,35 @@ public List<String> getColumnLabels()
             return fieldLabels;
         }
 
+        /**
+         * Extract label from header cell. Editable grid header cells have several different layouts. What they have in
+         * common is that the label is the first text node in the cell, possibly within a &lt;span&gt;
+         */
+        private String getLabelFromHeaderCell(WebElement el)
+        {
+            // Use text nodes to ignore browser whitespace formatting
+            List<String> textNodes = WebElementUtils.getTextNodesWithin(el);
+            if (textNodes.isEmpty())
+            {
+                List<WebElement> children = Locator.xpath("./*").findElements(el);
+                if (children.isEmpty())
+                {
+                    return ""; // probably the selection checkbox column
```

Comment:
I think the number of places where the editable grid has a checkbox column is few and far between.